include/dpp/cluster.h

```diff
@@ -232,6 +232,16 @@ class DPP_EXPORT cluster {
 	 */
 	cluster(const cluster&&) = delete;
 
+	/**
+	 * @brief dpp::cluster is non-copyable
+	 */
+        cluster& operator=(const cluster&) = delete;
+
+	/**
+	 * @brief dpp::cluster is non-moveable
+	 */
+        cluster& operator=(const cluster&&) = delete;
```

Comment:
The parameter shouldn't be const here

Reply:
Why?

Reply:
isnt it being changed? therefore never const?

Reply:
Because that's not a move assignment operator. A move assignment operator is a special function with a specific signature that takes a non-const rvalue.

Reply:
https://en.cppreference.com/w/cpp/language/move_assignment

Reply:
Seems fine after all, though I don't think it makes a lot of sense but since there isn't any practical issues with it i'll mark it resolved

Reply:
Nvm yeah that is quite the pog, must I say https://godbolt.org/z/6P7fPe1Gx 👍